src/main/resources/config/application-docker.yml

```diff
@@ -0,0 +1,15 @@
+# this profile contains the default variables for the docker setups
```

Comment:
```suggestion
# Artemis configuration profile with the default variables for the docker setups
```
With all the yaml files used for different things specifying a bit more explicitly that this should be part of an Artemis config and is not used by Docker compose directly might make it easier to understand for new people what the file contains.